tutorials/admin/getting-started/installation.md

```diff
@@ -1 +1,237 @@
-# installation
+# Installation
+
+This tutorial describes how to install a Marble node on a server. 
+
+## Prerequisites
+
+### Docker
+
+Marble is deployed as a [Docker](https://www.docker.com/) application using the 
+[compose](https://docs.docker.com/compose/) tool to manage and various services that make us the application.
+
+To install Docker on Unix/Linux either install [Docker Engine](https://docs.docker.com/engine/install/) (recommended) or
+[Docker Desktop](https://docs.docker.com/desktop/). If using an operating system other than Unix/Linux, you must use
```

Comment:
> If using an operating system other than Unix/Linux, you must use
Docker Desktop.

Consider putting this in a Note section.  But I understand if you don't want to clutter the screen with too many of these types of attention-getting text boxes.

Reply:
I think I'll keep it as is. The following warning section is meant as the attention grabbing warning.